db/migrate/20170512162050_create_terms_and_conditions.rb

```diff
@@ -0,0 +1,18 @@
+class CreateTermsAndConditions < ActiveRecord::Migration
+  def change
+    create_table :terms_and_conditions do |t|
+      t.string :name
+      t.string :title
+      t.text :text
+      t.string :version
+      t.boolean :latest, default: false
```

Comment:
Good point, I forgot the indexes altogether, although there will be so few of them that I suspect it hardly matters. The thing with latest is that since the versions may potentially be a string like "1.4" it's quite difficult to make those comparisons in a SQL query. I thought about doing it as the most recent `created_at` is the latest, but that could have flaws if we for some reason wanted to roll back to an earlier version. So yeah, it's like a cache. There's no logic for creating these things in the API yet, but I figure when we add that we could manage the state of the flag in that code.